todo-angular/src/app/todo/todolist/todolist.component.html

```diff
@@ -0,0 +1,11 @@
+<p>todolist works!</p>
+
+<add-new-todo></add-new-todo>
+
+<mat-list>
+<mat-list-item 
+  *ngFor="let tasks of todoListTasks"
```

Comment:
Better naming would be ="let task of todoListTasks"

Reply:
Weren't sure if Angular knew the difference between identical variable names, but seems so